data/fields/bicycle_parking.json

```diff
@@ -1,5 +1,15 @@
 {
     "key": "bicycle_parking",
     "type": "combo",
-    "label": "Type"
+    "label": "Type",
+    "strings": {
+        "options": {
+            "stands": "Stand (supports bicycle frame)",
+            "wall_loops": "Wheelbender (supports wheel only)",
```

Comment:
hmm, _wheelbender_ feels a little too colloquial… an opinion from a native speaker would be appreciated on this one. //cc @1ec5 @zekefarwell @ZeLonewolf

Reply:
AFAIK it is both colloquial and state-of-the-art term among people who care about bicycle parking types: cyclists.

Not sure is there even more professional term for that, and 

(the same situation is in Poland, "wyrwikółka" is colloquial and at the same time used by cyclists)

And wall loops refers to some specific subtype of them but nowadays is used for not only ones like this:

![screen](https://user-images.githubusercontent.com/899988/194605668-07c42d51-28b0-4c5f-a1ff-bfa2090532b3.png)

Reply:
I was not familiar with the term wheelbender, but it makes sense and is readily found with a Google search. I also don't know of a different term for this rack type.  However, it sounds like it refers to any rack that only supports the wheel, not just wall loops.
https://bikeswelcome.org/2016/12/04/wheel-benders-whats-all-the-fuss/